### PR TITLE
[DOC] pulsar web service executor metrics doc

### PR DESCRIPTION
**Pull request by tjiuming**
### Motivation

add doc of pulsar-web executor metrics

### Modifications

1. add doc in reference-metrics.md

### Does this pull request potentially affect one of the following parts:

*If `yes` was chosen, please highlight the changes*

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API: (no)
  - The schema: (no)
  - The default values of configurations: (no)
  - The wire protocol: (no)
  - The rest endpoints: (no)
  - The admin cli options: (no)
  - Anything that affects deployment: (no)

### Documentation

Check the box below or label this PR directly (if you have committer privilege).

Need to update docs? 

- [ ] `doc-required` 
  
  (If you need help on updating docs, create a doc issue)
  
- [ ] `no-need-doc` 
  
  (Please explain why)
  
- [x] `doc` 
  
  (If this PR contains doc changes)

### CODE REVIEW

**Comment by tjiuming** [comment]
this pr related to https://github.com/apache/pulsar/pull/14742

File: site2/docs/reference-metrics.md [hunk 1]
```diff
@@ -495,6 +496,22 @@ All the jetty metrics are labelled with the following labels:
 | jetty_stats_seconds | Gauge | Time in seconds stats have been collected for. |
 | jetty_responses_bytes_total | Counter | Total number of bytes across all responses. |
 
+### Web service executor metrics
+
+> For a functions-worker running separately from brokers, its Jetty metrics are only exposed when `includeStandardPrometheusMetrics` is set to `true`.
```
Comment (Anonymitaet) [inline site2/docs/reference-metrics.md:501]:
```suggestion
> For functions workers running separately from brokers, their Jetty metrics are only exposed when `includeStandardPrometheusMetrics` is set to `true`.
```

File: site2/docs/reference-metrics.md [hunk 2]
```diff
@@ -495,6 +496,22 @@ All the jetty metrics are labelled with the following labels:
...
+
+All the web service executor metrics are labelled with the following labels:
+
+- *cluster*: `cluster=${pulsar_cluster}`. `${pulsar_cluster}` is the cluster name that you have configured in the `broker.conf` file.
+
+| Name | Type | Description |
+|---|---|---|
+| pulsar_web_executor_max_threads | GAUGE | The max threads of pulsar-web  thread pool |
+| pulsar_web_executor_min_threads | GAUGE | The max threads of pulsar-web thread pool |
```
Comment (Anonymitaet) [inline site2/docs/reference-metrics.md:510]: the min threads

File: site2/docs/reference-metrics.md [hunk 3]
```diff
@@ -495,6 +496,22 @@ All the jetty metrics are labelled with the following labels:
...
+| pulsar_web_executor_idle_threads | GAUGE | The idle threads of pulsar-web thread pool |
+| pulsar_web_executor_active_threads | GAUGE | How many threads doing task of pulsar-web thread pool |
```
Comment (Anonymitaet) [inline site2/docs/reference-metrics.md:512]:
```suggestion
| pulsar_web_executor_active_threads | GAUGE | The number of threads performing tasks of pulsar-web thread pool |
```

File: site2/docs/reference-metrics.md [hunk 4]
```diff
@@ -495,6 +496,22 @@ All the jetty metrics are labelled with the following labels:
...
+| pulsar_web_executor_current_threads | GAUGE | How many threads in the pulsar-web thread pool now |
```
Comment (Anonymitaet) [inline site2/docs/reference-metrics.md:513]:
```suggestion
| pulsar_web_executor_current_threads | GAUGE | The number of threads in the pulsar-web thread pool |
```

**Review by Anonymitaet — APPROVED** [review]
LGTM from the tech writing pespective

**Review by codelipenghui — APPROVED** [review]

**Comment by codelipenghui** [comment]
@tjiuming Please help resolve the conflicts

**Comment by tjiuming** [comment]
@Anonymitaet  seems this pr is able to merge, otherwise will conflict  with master

**Comment by Anonymitaet** [comment]
fyi @momo-jun

**Pull request merged**